### ISSUE TITLE
Access properties without opening modal?

### ISSUE THREAD

**Issue by mdbraber** [mgsloan/todoist-shortcuts#197]
For a feature I'm trying to create I'd like to access some elements of the Project Comments. They are only accessible if I click and make the project comments visible via the modal. Is there a way to click it and prevent it from being visible so I can access the properties?

**Comment by mgsloan** [comment]
Hi!  You could probably use `addStyle` to hide the modal from the user
temporarily.  This is kinda tricky, though, because the code needs to be
very sure to remove the CSS once the property retrieval is done.

Another issue is that if this needs to be open for any longer than an
instant, it could interfere with user interaction

**Comment by mdbraber** [comment]
I'll investigate, but doesn't sound indeed like a tricky option. I'll let you know if I had any success.

**Comment by mgsloan** [comment]
Closing as wontfix
